jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/udp/psm/PsmReceiver.java

```diff
@@ -58,69 +59,40 @@ public void run() {
                     logger.debug("Packet received from {}:{}", senderIp, senderPort);
 
                     // extract the actualPacket from the buffer
-                    byte[] payload = removeHeader(packet.getData());
+                    byte[] payload = packet.getData();
                     if (payload == null)
                         continue;
-                    String payloadHexString = HexUtils.toHexString(payload);
-                    logger.debug("Packet: {}", payloadHexString);
+
+                    // convert bytes to hex string and verify identity
+                    String payloadHexString = HexUtils.toHexString(payload).toLowerCase();
```

Comment:
Please move this repetitive logic into lib or utils to allow reuse.

Reply:
We can also create an abstract class for the receivers. This will be part of our backlog task.